frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/CheckReAuthUserHandler.java

```diff
@@ -147,18 +147,29 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                     .orElseGet(() -> generateErrorResponse(emailUserIsSignedInWith, auditContext));
         } catch (AccountLockedException e) {
 
-            auditService.submitAuditEvent(
-                    FrontendAuditableEvent.AUTH_ACCOUNT_TEMPORARILY_LOCKED,
-                    auditContext,
-                    e.getErrorResponse() == ErrorResponse.ERROR_1045
-                            ? AuditService.MetadataPair.pair(
-                                    "number_of_attempts_user_allowed_to_login",
-                                    configurationService.getMaxPasswordRetries())
-                            : AuditService.MetadataPair.pair(
-                                    "number_of_attempts_user_allowed_to_login",
+            switch (e.getErrorResponse()) {
+                case ERROR_1057:
+                    auditService.submitAuditEvent(
+                            FrontendAuditableEvent.AUTH_REAUTH_INCORRECT_EMAIL_LIMIT_BREACHED,
+                            auditContext,
+                            AuditService.MetadataPair.pair(
+                                    "attemptNoFailedAt",
                                     configurationService.getMaxEmailReAuthRetries()));
```

Comment:
From the name of the pair, should this be the number of failed attempts rather than the maximum number of attempts allowed?

Reply:
See comment [here](https://github.com/govuk-one-login/authentication-api/pull/5144#discussion_r1736409895)

Reply:
We might be able to switch back to this approach when ongoing work on reauth is done, since we'll not delete counts.